source/server/listener_manager_impl.cc

```diff
@@ -698,7 +698,8 @@ void ListenerManagerImpl::startWorkers(GuardDog& guard_dog) {
   const auto listeners_pending_init =
       std::make_shared<std::atomic<uint64_t>>(workers_.size() * active_listeners_.size());
   for (const auto& worker : workers_) {
-    ENVOY_LOG(info, "starting worker {}", i);
+    i++;
+    ENVOY_LOG(debug, "starting worker {}", i);
```

Comment:
@mattklein123  PTAL. As discussed in the other PR. With out this change, it just logs "starting worker 0" for every worker. Also I have just downgraded to debug from info as this comes more if we have more concurrency. But if you think it needs to be at info, I can revert it. I think this was recently added, I have not seen this log line coming when Envoy starts earlier.